### ISSUE TITLE
Change the URL in the search

### ISSUE THREAD

**Issue by 0xsimulacra** [alexandermalcin/intellij-platform-browser-plugin#4]
Hello,

I open this issue related to this comment: https://plugins.jetbrains.com/plugin/23275-browser/reviews#review=95868-95929

Indeed as you can see from the video I post, the address bar is just no "clickable" in the sense that as soon as i click on the address bar, the text in it get selected then quicly get deselected and I cannot type anything no matter what.

Side note: Not really a problem, but If I was using the browser, I need to get out of it by clicking somewhere else int he editor so I can use my workspace keyboard shortcuts to switch between workspace (Ctrl+ Alt + UP/Down).

I'm running Goland that I installed on my openSuse linux using Toolbox. I don't think this has to do anything with the problem as all the plugins works just fine for me.

[Screencast from 2024-01-27 21-22-42.webm](https://github.com/alexandermalcin/intellij-platform-browser-plugin/assets/21297015/4bf1f2f0-8e2c-4b3a-a2c4-f8f78e2e8cd7)

**Comment by alexandermalcin** [comment]
Oh, the plugin is really unusable in your environment.

Many thanks for your effort! I'll take care of this ASAP. 
Unfortunately, I can't tell you at the moment when I'll be ready. Firstly, because I don't have Linux and secondly, I have to do some things beforehand. 

But maybe I can solve it another way. 
I'll keep you updated defensively.

Thanks for your input and sorry for the inconvenience!
This error is annoying.

**Comment by alexandermalcin** [comment]
Which version of Goland are you using? Latest?

**Comment by 0xsimulacra** [comment]
Yes, I'm using the latest version: 2023.3.3. The problem existed for me even in the version before.

The side note that I've added in the issue, one extention of that wierd behaviour is that sometimes I can't even get the editor to focus on the code so I can type, I have to completly hide or shutdown the Browser plugin so I can type code again, and this is really annoying. 

In my Linux, I also using X11 as my displkay server, not even Wayland.

Thank you for your quick response. Ofc, I understand, you can focus on the things you have t do beforehand and maybe the error will solve itself because of that :)

**Comment by MichaelDemeyer** [comment]
Hello,

I have the same issue here in PhpStorm on Ubuntu.
A workaround seems to be opening and closing the DevTools ToolWindow.
Maybe this can also help in fixing the bug ?

Thx for the great plugin btw
